README.md

```diff
@@ -202,7 +202,7 @@ org.freedesktop.tuhi1.Device
       signal arriving, the property DrawingsAvailable may still be updated
       and it's the responsibility of the client to fetch the JSON data.
 
-  Method: GetJSONData(index: u) -> (s)
+  Method: GetJSONData(index: t) -> (s)
       Returns a JSON file with the drawings specified by the index argument.
       Drawings are zero-indexed and the requested index must be less than
       the DrawingsAvailable property value. See section JSON FILE FORMAT for
```

Comment:
You seem to have forgotten to update this description in the README. A timestamp is not an index :)

Reply:
Fixed as ef8747f0328c9c7c26cec0f73017d5a661ee5b2d, thanks